src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +172,33 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::get_output_element_type(const Napi::CallbackInfo& info) {
+    if(info.Length() != 1 || !info[0].IsNumber()) {
+        reportError(info.Env(), "Invalid argument. Expected a single number for output index.");
+        return info.Env().Undefined();
+    }
+
+    try {
+        auto idx = info[0].As<Napi::Number>().Int32Value();
+        auto output = _model->output(idx);
+        auto element_type = output.get_element_type();
```

Comment:
[Here's](https://github.com/openvinotoolkit/openvino/blob/b8ba9038d63cb933f81655326d23ba7a30421c4a/src/bindings/js/node/include/helper.hpp#L96)  helper to convert ov::element::Type_t  to Napi::String